packages/ovh-manager/ovh-manager.js

```diff
@@ -92,4 +99,46 @@ angular
     _.set(OvhHttpProvider, 'returnSuccessKey', 'data'); // By default, request return response.data
     _.set(OvhHttpProvider, 'returnErrorKey', 'data'); // By default, request return error.data
   })
-  .config(routing);
+  .config(routing)
+  .service('ManagerNavbarService', navbarService)
+  .service('NavbarNotificationService', notificationService)
+  .run(($transitions,
```

Comment:
move it away from mainfile :)

split split split, else the mainfile will become a trash file :/